src/conversions.jl

```diff
@@ -434,6 +434,39 @@ julia> AstroTime.Epochs.ut1tt(ut1.jd1, ut1.jd2, AstroTime.Epochs.deltat(ut1))
      date, date1
 end
 
+
+"""
+    tdbtcb(jd1, jd2)
+
+Transform a two-part Julian date from `TDB` to `TCB`.
+
+# Example
+
+julia> tdb = Epoch{TDB}(2.4578265e6, 0.30440190993249416)
+2017-03-14T07:18:20.325 TDB
+julia> AstroTime.Epochs.tdbtcb(tdb.jd1, tdb.jd2)
+(2.4578265e6, 0.30440190993249416)
+```
+"""
+@inline function tdbtcb(jd1, jd2)
+    const t77td = MJD + MOD_JD_77
```

Comment:
You don't `const` annotations in function, they only work in global scope. I think this will be an error in Julia 1.0.